packages/client/hmi-client/src/types/Types.ts

```diff
@@ -967,6 +967,7 @@ export enum ProvenanceType {
     Artifact = "Artifact",
     Code = "Code",
     Document = "Document",
+    Workflow = "Workflow",
```

Comment:
I think this change is supposed to be done in `AssetType.java` instead. That'll fix the linting error.